proactive-support-bot.py

```diff
@@ -39,6 +42,10 @@ def print_help(self, file=None):
         raise HelpRequest({'parser': self})
 
 
+def dashboard_uri(cluster):
+    return 'https://telemeter-lts-dashboards.datahub.redhat.com/d/kSomLV7Wk/pre-production-cluster-by-id?orgId=1&var-_id={}'.format(cluster)
```

Comment:
probably need to make this configurable.  Also, it leaks an internal hostname.

Reply:
We can load it from an env var

Reply:
Yup, something low-tech like that would be fine.

Reply:
Fixed with 20d9a7f -> e4ad415, which also rebases onto the new master.